src/adagrams.js

```diff
@@ -1,8 +1,204 @@
+import _, { result, shuffle } from 'underscore'; 
+
+const printValue = (value, letter) => {
```

Comment:
This should probably be called something like `repeatLetter` instead of `printLetter` (since it doesn't print anything out).